jboss-kie-kieserver/added/openshift-launch.sh

```diff
@@ -31,6 +31,7 @@ CONFIGURE_SCRIPTS=(
   $JBOSS_HOME/bin/launch/maven-settings.sh
   $JBOSS_HOME/bin/launch/access_log_valve.sh
   $JBOSS_HOME/bin/launch/filters.sh
+  $JBOSS_HOME/bin/launch/jboss-kie-common.sh
```

Comment:
I don't think you need to add it here, the jboss-kie-common script only have functions to be called, it only needs to be sourced when needs to be used.